src/components/mapgl/mapgl.component.ts

```diff
@@ -851,7 +865,12 @@ export class MapglComponent implements OnInit, AfterViewInit, OnChanges {
       const rawSouthOffset = bottomLeftOffsetLatLng.lat;
       const rawEastOffset = topRghtOffsetLatLng.lng;
       const rawNorthOffset = topRghtOffsetLatLng.lat;
-
+      const visibleLayers = new Set<string>();
+      this.visualisationsSets.status.forEach((b, vs) => {
+        if (b) {
```

Comment:
same comment for the filter

Reply:
In order to make a future rebase easier, I will treat this in the next PR